packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -380,7 +380,7 @@ export class FetchInstrumentation extends InstrumentationBase<
               plugin._addHeaders(options, url);
               plugin._tasksCount++;
               return original
-                .apply(this, [url, options])
+                .apply(this, options instanceof Request ? [options] : [url, options])
```

Comment:
same here. we should pass whatever the user gave us

Reply:
Won't that break the functionality in `_addHeaders` if they didn't pass any options in the first place (just url)?  I agree with the previous "no span -> call the original just as originally called".

Reply:
ah you're right